src/js/components/AppVersionListComponent.jsx

```diff
@@ -77,18 +122,20 @@ var AppVersionListComponent = React.createClass({
   },
 
   getAppVersionTable: function () {
+    var state = this.state;
+
     // take out current version, to be displayed seperately
-    var appVersions = this.props.app.versions.models.slice(1);
+    var appVersions = state.appVersions.slice(1);
 
     var loadingClassSet = classNames({
       "text-muted text-center": true,
-      "hidden": this.props.fetchState !== States.STATE_LOADING
+      "hidden": state.fetchState !== States.STATE_LOADING
     });
 
     var errorClassSet = classNames({
       "text-danger text-center": true,
-      "hidden": this.props.fetchState === States.STATE_LOADING ||
-        this.props.fetchState === States.STATE_SUCCESS
+      "hidden": state.fetchState === States.STATE_LOADING ||
+        state.fetchState === States.STATE_SUCCESS
```

Comment:
Yeah, why not, also possible. :)
But I think I will let it like it is -> 2 positive cases what hides the false case.
Else we would have:
1 negative or 1 positive case what hides the false case.
I prefer the first.